code/datums/status_effects/neutral.dm

```diff
@@ -47,25 +47,91 @@
 
 /datum/status_effect/high_five
 	id = "high_five"
-	duration = 5 SECONDS
+	duration = 10 SECONDS
 	alert_type = null
+	status_type = STATUS_EFFECT_REFRESH
+	var/critical_success = "high-five EPICALLY!"
+	var/success = "high-five!"
+	var/request = "requests a high-five."
+	var/obj/item/item_path = /obj/item/latexballon
+	var/sound_effect = 'sound/effects/snap.ogg'
+
+/// So we don't leave folks with god-mode
+/datum/status_effect/high_five/proc/wiz_cleanup(mob/user, mob/highfived)
+	user.status_flags &= ~GODMODE
+	highfived.status_flags &= ~GODMODE
+
+/datum/status_effect/high_five/on_apply()
+	if(!istype(owner, /mob/living/carbon))
+		return FALSE
+	. = ..()
+
+	var/mob/living/carbon/user = owner
+	for(var/mob/living/carbon/C in orange(1))
+		if(C.has_status_effect(type) && C != user)
+			if(iswizard(user) && iswizard(C))
```

Comment:
do the `iswizard(user)` outside of the for loop and store it in a variable, it will skip you having to perform redundant type checks